src/client/client.go

```diff
@@ -4,14 +4,15 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	args2 "github.com/newrelic/nri-rabbitmq/src/args"
 	"net/http"
 	"os"
 	"strings"
 	"time"
 
 	nrHttp "github.com/newrelic/infra-integrations-sdk/http"
 	"github.com/newrelic/infra-integrations-sdk/log"
+
+	args2 "github.com/newrelic/nri-rabbitmq/src/args"
```

Comment:
args2?

Reply:
I add it like that for consistance since it is called like that in all over the integration, i guess due to conflict with sdk args packages.